src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search__bar">
+      
+      <form class="search__bar big__bar" data-qa="big"> 
+        <input 
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="search__input big__input"
+          data-qa="keypress"
+          >
+        <img 
+          src="images/Search.svg" 
+          alt="search icon" 
+          class="big__logo"
+        >
+      </form>
+
+      <form class="search__bar small__bar" data-qa="small"> 
```

Comment:
Just use modifiers --small to change styles of block/element
```suggestion
      <form class="search-bar search-bar--small" data-qa="small"> 
```